internal/test/wmcb_test.go

```diff
@@ -200,11 +201,20 @@ func TestWMCBUnit(t *testing.T) {
 	// Forcefully close it so that we can execute the binary later
 	dstFile.Close()
 
+	stdout := os.Stdout
+	r, w, _ := os.Pipe()
+	os.Stdout = w
+
 	// Remotely execute the test binary.
-	_, err = framework.winrmClient.Run(remotePowerShellCmdPrefix+framework.remoteDir+"\\"+"wmcb_unit_test.exe",
+	_, err = framework.winrmClient.Run(remotePowerShellCmdPrefix+framework.remoteDir+"\\"+
+		"wmcb_unit_test.exe --test.v",
 		os.Stdout, os.Stderr)
-	// TODO: Get the stdout, stderr and use assertions here
 	assert.NoError(t, err, "error while executing the test binary remotely")
+	// TODO: Get the stdout, stderr and use assertions here
```

Comment:
Do you still need this comment?

Reply:
Nope, you're right. Thanks for catching it.